charts/openebs/values.yaml

```diff
@@ -252,17 +251,17 @@ featureGates:
   UseOSDisk:
     enabled: false
     featureGateFlag: "UseOSDisk"
-
-crd:
-  enableInstall: true
```

Comment:
@akhilerm Maya-apiserver deployment uses the variable , we have to keep it.

Reply:
Done.

Reply:
good catch @prateekpandey14 !